### PR TITLE
chore: bump vite to 8.1.3

### PR DESCRIPTION
**Pull request by hi-ogawa**
### Description

Isolate whether the released Vite version jump (8.0.11 -> 8.1.3) alone causes browser-mode stacktrace column drift, independent of the SSR transform change tested in https://github.com/vitest-dev/vitest/pull/10722

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. If the feature is substantial or introduces breaking changes without a discussion, PR might be closed.
- [ ] Ideally, include a test that fails without this PR but passes with it.
- [ ] Please, don't make changes to `pnpm-lock.yaml` unless you introduce a new test example.
- [ ] Please check [Allow edits by maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to make review process faster. Note that this option is not available for repositories that are owned by Github organizations.

### Tests
- [ ] Run the tests with `pnpm test:ci`.

### Documentation
- [ ] If you introduce new functionality, document it. You can run documentation with `pnpm run docs` command.

### Changesets
- [ ] Changes in changelog are generated from PR name. Please, make sure that it explains your changes in an understandable manner. Please, prefix changeset messages with `feat:`, `fix:`, `perf:`, `docs:`, or `chore:`.

### CODE REVIEW

File: test/browser/specs/errors.test.ts [hunk 1]
```diff
@@ -164,40 +163,21 @@ test('prints source-mapped stack for optimized dependency', async () => {
 
   for (const [name, tree] of Object.entries(projectTree)) {
     if (name === 'webkit') {
-      if (rolldownVersion) {
-        expect(tree).toMatchInlineSnapshot(`
```
Comment (hi-ogawa) [inline test/browser/specs/errors.test.ts:168]: This looks like technically stack trace (source map) regressed for webkit. Should flag it to Vite / Oxc though may be edge case.
